src/components/Claim/Claim.consts.ts

```
@@ -17,6 +18,12 @@ export const INIT_VIEW_STATE: IClaimScreenState = {
 
 export const CLAIM_SCREEN_FLOW: ClaimScreens[] = ['INITIAL', 'CONFIRM', 'SUCCESS']
 
+export interface IFlowManagerClaimComponents {
+    INITIAL: ({}: IClaimScreenProps) => {},
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)